routes/index.js

```diff
@@ -96,6 +94,19 @@ exports.conversation = function (req, res) {
 	});
 }
 
+exports.account = function (req, res){
+	if (!req.user || !req.user._id) {
+		res.render('error', {
+			title: "Looks like you're not logged in!"
+		});
+		return;
+	} else{
+		res.render('account', {
+			user: req.user
+		});
+	}
+}
```

Comment:
Ah, after looking at more of the PR below, perhaps this handles errors when the user accesses the page where he/she chooses their new password?